client/executor/src/wasm_runtime.rs

```diff
@@ -149,12 +152,12 @@ const MAX_RUNTIMES: usize = 2;
 /// the memory reset to the initial memory. So, one runtime instance is reused for every fetch
 /// request.
 ///
-/// The size of cache is equal to `MAX_RUNTIMES`.
+/// The size of cache is configurable via the cli option `--runtime-cache-size`.
 pub struct RuntimeCache {
 	/// A cache of runtimes along with metadata.
 	///
 	/// Runtimes sorted by recent usage. The most recently used is at the front.
```

Comment:
Comment could use an update

Reply:
This comment is still true, LruCache "sorts" items by recent usage, the most recently used still at the front.